nixos/modules/services/databases/arangodb.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.arangodb;
+
+in {
+
+  ###### interface
+
+  options = {
+    services.arangodb = {
+      enable = mkEnableOption "ArangoDB server";
+
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/db/arangodb";
+        description = "ArangoDB data dir";
+      };
+    };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+    users.extraUsers.arangodb = {
+      name = "arangodb";
+      uid = config.ids.uids.arangodb;
+      group = "arangodb";
+      description = "ArangoDB server user";
+    };
+
+    users.extraGroups.arangodb.gid = config.ids.gids.arangodb;
+
+    systemd.services.arangodb = {
+      description = "ArangoDB server";
+
+      wantedBy = [ "multi-user.target" ];
+
+      after = [ "network.target" ];
+
+      preStart = ''
+        mkdir -p ${cfg.dataDir}
+        chown -R arangodb:arangodb ${cfg.dataDir}
+        '';
+
+      script = ''
+        exec ${pkgs.arangodb}/bin/arangod \
+            --database.directory ${cfg.dataDir} \
+            --javascript.app-path ${cfg.dataDir} \
+            --log.file ${cfg.dataDir}/arangodb.log \
```

Comment:
Is there a way to redirect output to standard system log (`journald`) as this is the standard in NixOS and preferred.

If there is no way to redirect output to `journald` instead of a log file does `arangodb` automatically rotate this log file?